Mathlib/LinearAlgebra/TensorProduct/Vanishing.lean

```diff
@@ -274,8 +274,9 @@ theorem rTensor_injective_of_forall_fg_rTensor_injective
   (forall_vanishesTrivially_iff_forall_rTensor_injective R).mp
     ((forall_vanishesTrivially_iff_forall_fg_rTensor_injective R).mpr hMN) M'
 
-@[deprecated (since := "2025-01-03")] alias forall_vanishesTrivially_iff_forall_FG_rTensor_injective
-  := forall_vanishesTrivially_iff_forall_fg_rTensor_injective
+@[deprecated (since := "2025-01-03")] alias
+forall_vanishesTrivially_iff_forall_FG_rTensor_injective :=
```

Comment:
```suggestion
@[deprecated (since := "2025-01-03")]
alias forall_vanishesTrivially_iff_forall_FG_rTensor_injective :=
```
I think that this formatting is more common.